### DISCUSSION TITLE
Big file in repo?

### DISCUSSION THREAD

**Discussion by T-Dynamos** [termux-user-repository/tur#38] [General]
/disabled-tur/libllvm-15/flang-new.tar.gz
This file makes repo larger. Can we remove it?

@licy183

**Comment by 2096779623** [comment]
Maybe not needed.

**Comment by licy183** [comment]
Sorry for that. Actually, we can remove it from the repo, but the commit history won't remove it.

**Comment by licy183** [comment]
I will disable the `protected branch`, force update and re-enable it. If you have a fork, please rebase later. Sorry for that again!
